Extensions/Cosmos/Cosmos.DataTransfer.CosmosExtension/CosmosSettingsBase.cs

```diff
@@ -11,7 +11,7 @@ public abstract class CosmosSettingsBase : IValidatableObject
         [Required]
         public string? Container { get; set; }
         public ConnectionMode ConnectionMode { get; set; } = ConnectionMode.Gateway;
-
+        public string? WebProxy { get; set; }
```

Comment:
@admsteck, can you update the [README.md](https://github.com/AzureCosmosDB/data-migration-desktop-tool/blob/main/Extensions/Cosmos/README.md) with information about this new setting?

Reply:
Added